src/client/fuse_ll.cc

```diff
@@ -161,6 +161,18 @@ static void fuse_ll_lookup(fuse_req_t req, fuse_ino_t parent, const char *name)
   UserPerm perms(ctx->uid, ctx->gid);
   GET_GROUPS(perms, req);
 
+  if (!i1)
+  {
+    r = cfuse->client->lookup_ino(parent, perms, &i1);
+    if (r < 0) {
+      fuse_reply_err(req, -r);
+      return;
+    }
+    else {
+      name = ".";
```

Comment:
don't touch name.   the name can be "." or ".."

Reply:
@ukernel Done. Please check now. Thanks.